src/platforms/web/runtime/components/transition-group.js

```diff
@@ -77,6 +77,7 @@ export default {
       for (let i = 0; i < prevChildren.length; i++) {
         const c: VNode = prevChildren[i]
         c.data.transition = transitionData
+        c.data.isDisplayNone = c.elm.style.display === 'none'
```

Comment:
Accessing element.style here in a loop will probably cause serious perf issues due to forced layouts. More info: https://developers.google.com/web/fundamentals/performance/rendering/avoid-large-complex-layouts-and-layout-thrashing
It's probably better to just check for empty rects (` { x: 0, y: 0, width: 0, height: 0, top: 0, right: 0, bottom: 0, left: 0 }`) below